devops/docker/docker-compose.yml

```diff
@@ -80,5 +80,18 @@ services:
     environment:
       - "SPRING_PROFILES_ACTIVE=test"
 
+  db2:
```

Comment:
If you don't mind, can you please add this before the services along with the other db's? 🙂 
Just to keep it orderly.

Reply:
Yup, Updated the order.